Release-Process.md

```diff
@@ -84,8 +84,17 @@
     BUILD_HEAD=false
     OBS_BRANCH="the-kata-branch-that-is-release"
     ```
+    Note: if the release is `Alpha` , `Beta` , or `RC` (that is part of a `master` relase), please use `OBS_BRANCH=master`
```

Comment:
`.` at the end of the line and please also add a blank line.

Reply:
s/relase/release/

Reply:
- s/if/If/
- Can you format this as a standard "[Note](https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#notes)":

  > **Note:** If the release ...